packages/cli/src/modules/chat-hub/chat-hub.service.ts

```diff
@@ -806,24 +837,36 @@ export class ChatHubService {
 				],
 			},
 			[NODE_NAMES.RESTORE_CHAT_MEMORY]: {
-				main: [[{ node: NODE_NAMES.AI_AGENT, type: NodeConnectionTypes.Main, index: 0 }]],
+				main: [
+					[
+						{ node: NODE_NAMES.REPLY_AGENT, type: NodeConnectionTypes.Main, index: 0 },
+						{ node: NODE_NAMES.TITLE_GENERATOR_AGENT, type: NodeConnectionTypes.Main, index: 0 },
```

Comment:
Title generator doesn't need memory, but having it here ensures that the reply agent is executed earlier (otherwise the first chunk of streaming is delayed)